src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -2899,11 +2872,13 @@ internal Binder GetBinder(CSharpSyntaxNode syntaxNode)
             return this.DeclaringCompilation.GetBinder(syntaxNode);
         }
 
-        private static void MergePartialMembers(
-            ArrayBuilder<string> memberNames,
-            Dictionary<string, ImmutableArray<Symbol>> membersByName,
+        private void MergePartialMembers(
```

Comment:
Consider adding a remark to this method that `membersByName` should only be a ref to a local, or corruption could result.

Reply:
> Consider adding a remark to this method that membersByName should only be a ref to a local, or corruption could result.

The statement in the suggestion is false.

---
In reply to: [568865128](https://github.com/dotnet/roslyn/pull/50961#discussion_r568865128) [](ancestors = 568865128)